src/lib.rs

```diff
@@ -650,7 +656,15 @@ unsafe impl<T: Send> Sync for EventListener<T> {}
 
 impl<T> fmt::Debug for EventListener<T> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.write_str("EventListener { .. }")
+        let mut ds = f.debug_struct("EventListener");
+
+        if self.is_listening() {
+            ds.field("Listening", &"Yes");
+        } else {
+            ds.field("Listening", &"No");
+        }
```

Comment:
Instead of `yes` and `no`, I would just use `true` and `false`, which can be done by just using `self.is_listening()` as the field here.